core/controllers/android_e2e_config_test.py

```diff
@@ -76,20 +75,23 @@ def test_check_if_topic_is_published(self):
         story.validate()
         topic.validate(strict=True)
         exploration.validate(strict=True)
-        for node in story.story_contents.nodes:
-            self.assertTrue(node.exploration_id == EXPLORATION_ID)
+
+    def test_initialize_structure_thumbnails_exists(self):
+        self.post_json('/initialize_android_test_data', {
+        }, use_payload=False, csrf_token=None, return_json=False)
+        topic = topic_fetchers.get_topic_by_name('Android test')
+        story = story_fetchers.get_story_by_url_fragment(
+            'android-end-to-end-testing')
         self.get_custom_response(
             '/assetsdevhandler/topic/%s/assets/thumbnail/test_svg.svg' %
             topic.id, 'image/svg+xml')
         self.get_custom_response(
             '/assetsdevhandler/story/%s/assets/thumbnail/test_svg.svg' %
             story.id, 'image/svg+xml')
 
-        self.assertTrue(topic_rights.topic_is_published)
-
     def test_exploration_assets_are_loaded(self):
-        self.post_req(
-            '/initialize_android_test_data')
+        self.post_json('/initialize_android_test_data', {
```

Comment:
Here & elsewhere: this line wrapping seems weird to me. I'd expect to normally break at the ``(`` not the ``{``. I suggest updating these places for consistency with the rest of the codebase.

Reply:
Done